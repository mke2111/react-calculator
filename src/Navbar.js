import React from 'react';
import './index.css';

const Navbar = () => (
  <div className="nav">
    <a href="./">Home</a>
    <a href="./App">Calculator</a>
    <a href="./quote">Quotes</a>
  </div>
);

export default Navbar;
