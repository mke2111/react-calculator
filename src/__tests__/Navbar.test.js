import React from 'react';
import renderer from 'react-test-renderer';
import Navbar from '../Navbar';

it('test the render of the Navbar page', () => {
  const navbar = renderer
    .create(<Navbar />)
    .toJSON();
  expect(navbar).toMatchSnapshot();
});
