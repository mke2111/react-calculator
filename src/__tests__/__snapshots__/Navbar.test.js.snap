// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test the render of the Navbar page 1`] = `
<div
  className="nav"
>
  <a
    href="./"
  >
    Home
  </a>
  <a
    href="./App"
  >
    Calculator
  </a>
  <a
    href="./quote"
  >
    Quotes
  </a>
</div>
`;
